lib/pdk/module/update.rb

```diff
@@ -90,6 +90,7 @@ def fetch_remote_version(version)
 
         return version if data.nil?
 
+        # TODO: Handle branches with /'s in the name.
```

Comment:
Conventionally using `#` as a separator is the way forward here, but I haven't looked deep enough into the code here whether that's feasible here.